client/src/api/index.js

```diff
@@ -174,7 +110,14 @@ export const Visitors = {
 };
 
 export const CbAdmin = {
-  get: () => axios.get('/community-businesses/me'),
+  get: async () => {
+    const [cbRes, userRes] = await Promise.all([
+      axios.get('/community-businesses/me'),
```

Comment:
This method is in use:
https://github.com/TwinePlatform/twine-visitor/blob/7640e9ccb32c1379feae8c6a6d6fd2c465e1bbb5/client/src/cb-admin/pages/Dashboard.js#L51-L54

I really don't think we should be making other pages make an extra api call to simplify calling code in another.

Reply:
I'm not saying I don't want to do this, I'm just saying I don't really want to do this in _this_ PR. It can certainly be done in the very next PR. But I'd prefer not to make this one any bigger.

Reply:
Okay, can you make an issue or mark this with a `// TODO`

Reply:
Made #491